src/ucp/api/ucp_def.h

```diff
@@ -101,7 +101,10 @@ typedef enum {
     UCP_ERR_HANDLING_MODE_NONE,             /**< No guarantees about error
                                              *   reporting, imposes minimal
                                              *   overhead from a performance
-                                             *   perspective */
+                                             *   perspective. @note In this mode,
+                                             *   @ref ucp_ep_params_t::err_handler
+                                             *   is not used for error reporting.
```

Comment:
@evgeny-leksikov this means that no errors (of any kind) will be reported to the user, correct?

Reply:
No, user can get error status by return code.

Reply:
Or by request callback, but not callback associated with EP

Reply:
i meant reported through the err handle callback.

Reply:
This could be a little more clear that error reporting may still happen, just not with the EP callback. If a user is not as familiar with the API, it sounds a bit like no error reporting will happen at all. "@note In this mode, any error reporting will not generate calls to @ref ucp_ep_params_t::err_handler"